docs-user/async-posix-signal-control.md

```diff
@@ -0,0 +1,102 @@
+
+# Async POSIX Signal Control
+
+The Firefox profiler supports the use of asynchronous POSIX signals for a
+limited number of operations, specifically starting and stopping the profiler.
+This can be useful in situations where the normal UI or keyboard shortcut
+control of the profiler is not available.
+
+In brief, a user can send the POSIX signal `SIGUSR1` to the main Firefox process
+to start the profiler, and the signal `SIGUSR2` to stop the profiler. Upon
+recieiving the "stop" signal, the profiler will collect the current profile data
+and write it to a `.json` file in the user's downloads directory. For example,
+on MacOS:
+
+    // Send signal USR1 to the first (main) process of Firefox.app to start the profiler
+    kill -s USR1 `pgrep -f Firefox.app | head -n 1`
+    // wait for firefox to gather some data
+    sleep 10
+    // Send signal USR2 to the same process to stop the profiler
+    kill -s USR2 `pgrep -f Firefox.app | head -n 1`
+    // Find the result in the download folder
+    ls ~/Downloads/ | grep "profile.*json"
+
+From there, the user can open the written profile using the standard profiler
+UI, by navigating to [profiler.firefox.com](profiler.firefox.com), and selecting
+"Load a profile from file".
+
+Typically, sending a signal to a program is done using the `kill`
+[command](https://man7.org/linux/man-pages/man1/kill.1.html), as seen in the
+above example. It is important to specify the correct signal, as the default is
+`TERM`, which will terminate the program. For the profiler, only signals
+`SIGUSR1` (for starting) and `SIGUSR2` (for stopping) should be used. Note that they
+are written without the `SIG` prefix when used as an argument to `kill` i.e.:
+
+    kill -s USR1 <firefox pid>      // start the profiler
+    kill -s USR2 <firefox pid>      // stop the profiler
+
+    // Alternative syntax
+    kill -USR1 <firefox pid>
+    kill --signal USR1 <firefox pid>
+
+Only the process ID of the "main" process should be used. Child processes are
```

Comment:
nit: It could be nice to append this paragraph with `!>` here to make it a warning. See an example here: https://deploy-preview-5013--perf-html.netlify.app/docs/#/./guide-getting-started

This is fine like this too, but I thought adding it some colors would be nice.